app/src/main/java/com/samco/trackandgraph/database/DatabaseMigrations.kt

```diff
@@ -388,6 +388,13 @@ val MIGRATION_40_41 = object : Migration(40, 41) {
     }
 }
 
+val MIGRATION_41_42 = object : Migration(41, 42) {
```

Comment:
Sorry you will have a merge conflict here now because I used 42 to for a recent change. You should just need to change from 42 to 43 though.